modules/addStaticOnLocalStorage.js

```diff
@@ -0,0 +1,117 @@
+import { usersSettings } from './usersSettings.js'
+
+// Добавляет статистику по загрузкам в localStorage
+function addStaticOnLocalStorage() {
+    let elem = document.querySelector("#b-container > div.b-popup-wrapper.js-popup-wrapper.js-popup-wrapper--upload > div.b-popup.b-popup--blue.b-popup--upload.js-popup--upload")
+
+    let promise = new Promise((resolve, reject) => {
+        let observer = new MutationObserver((mutationRecords) => {
+            if (mutationRecords.length > 1) {
+                let saveScanStat = document.querySelector("#b-container > div.b-popup-wrapper.js-popup-wrapper.js-popup-wrapper--upload > div.b-popup.b-popup--blue.b-popup--upload.js-popup--upload.b-popup--edit.js-popup--edit > form > div.b-popup-button.js-upload-submit"); // кнопка по которой срабатывает скрипт
+                let q = document.querySelector("#chrono_list_content > div.b-chrono-items-container.js-chrono-items-container > div").childElementCount  //Кол-во элементов для фикла
```

Comment:
you can use such short names for large functions or classes, where another context gives you the entire impression, but here it's tricky to understand